src/Storage/Repository.php

```diff
@@ -6,6 +6,7 @@
 use Bolt\Events\StorageEvent;
 use Bolt\Events\StorageEvents;
 use Bolt\Storage\Entity\Builder;
+use Bolt\Storage\Entity\Entity;
```

Comment:
It gets the PHPDoc in-line so that when you're extending or typehinting, it doesn't show as an error in the IDE